docs/src/docs/cli/installation.md

```diff
@@ -2,101 +2,144 @@
 import vGaTrack from '@theme/directives/ga'
 </script>
 
-# Installation Guide for `kit` CLI
+# Installing Kit
 
-## Installation from GitHub Releases
-Welcome to the installation guide for the `kit`! This guide is designed to help you quickly and easily install the `kit` on your machine.
+This page includes instructions for:
 
-### Step 1: Downloading the `kit`
+* Installing from [MacOS packages](#🍎-macos-package-download)
```

Comment:
Should we call these $Platform Installation

Reply:
Sorry, I don't understand what you mean - do you mean the title should be `MacOS installation` rather than `Installing from MacOS packages`?

Reply:
yes, and same for the following platforms. Windows, Linux